pkg/statdb/statdb_test.go

```diff
@@ -115,6 +117,7 @@ func testDatabase(ctx context.Context, t *testing.T, sdb statdb.DB) {
 				AuditSuccessCount:  tt.auditSuccessCount,
 				UptimeCount:        tt.uptimeCount,
 				UptimeSuccessCount: tt.uptimeSuccessCount,
+				Meta:               pb.NodeOperator{},
```

Comment:
Same here - unnecessary initialization to the default empty value.

Reply:
Done 👍

Reply:
I still see `Operator:           pb.NodeOperator{},` in a few places.